### ISSUE TITLE
terms having the same name are used for different subject types

### ISSUE THREAD

**Issue by baskaufs** [tdwg/cd#223]
There are several terms where the same local name (or "Title" in the Google sheet) is used for terms under different class categories. For example, `identifier`, `source`, and `type` are listed under the classes CollectionDescriptionIdentifier, InstitutionIdentifier, and OrganisationalUnitIdentifier. Should these be changed to be different terms with different local names (i.e. collectionIdentifier, institutionIdentifier, organisationIdentifier, etc.) so that they would be unambiguous in a flat table (like a spreadsheet)? Or are we assuming that records are normalized enough that the same term could be used in different tables?  

Already separate terms have been used in some cases like startDate and statusStartDate. There are also some existing different type terms like digitalRecordType, objectType, and others. 

Whatever the decision is, it should be applied consistently. 

Before I noticed the repeated values, I recommended adopting dcterms:identifier, dcterms:source, and rdf:type. However, if there will be separate terms, we can't adopt those terms for all. But perhaps one could noted that the specific properties are essentially subproperties of these well-known terms.

**Comment by rondlg** [comment]
For sure less ambiguous names are helpful were they are new to the domain. In the cases were a field is used in the same strict sense as that used in another standard, I'd say those are a good case to stay the same. Thoughts?

**Comment by mswoodburn** [comment]
I think those have a good case if they're also unambiguous within the context of the CD standard, but would agree with Steve that duplicate property names can be a pain when working with flattened files, and reports, queries etc. I'd be inclined to make sure that all properties have unique names within the CD namespace, and use the subproperty approach where they map to more generic terms like dcterms:identifier.

**Comment by rondlg** [comment]
Absolutely, no two fields within the standard should have the same name.

**Comment by dbloom** [comment]
Agreed. No dupes.
